### PR TITLE
Dxdao bridger testnets

### PR DESCRIPTION
**Pull request by brunitob**

### CODE REVIEW

File: data/actions/L1HopBridger.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "networks": [5],
+    "addresses": ["0x853acb808357AD416cD9A73E2c99B2944cDB36db"],
+    "icon": "https://v2-mimic.s3.amazonaws.com/bridge.png",
+    "title": "Bridge",
+    "description": "Bridge assets from Goerli to Mumbai",
```
Comment (facuspagnuolo) [inline data/actions/L1HopBridger.json:6]: Should we use past tenses here? 

```suggestion
    "description": "Bridged assets from Goerli to Mumbai",
```

File: data/actions/L1HopBridger.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    "successMessage": "Assets bridged to the other side successfully"
```
Comment (facuspagnuolo) [inline data/actions/L1HopBridger.json:7]:
```suggestion
    "successMessage": "Assets bridged to the destination chain successfully"
```

File: data/actions/L2HopBridger.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "networks": [80001],
+    "addresses": ["0x563F70db0567b44644fd901Cd17B48A236b43849"],
+    "icon": "https://v2-mimic.s3.amazonaws.com/bridge.png",
+    "title": "Bridge",
+    "description": "Bridge assets from Mumbai to Goerli",
+    "successMessage": "Assets bridged to the other side successfully"
```
Comment (facuspagnuolo) [inline data/actions/L2HopBridger.json:7]:
```suggestion
    "successMessage": "Assets bridged to the destination chain successfully"
```

File: data/smart-vaults/dxdao-bridger-l2.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+    "networks": [80001],
+    "addresses": ["0x282d45DFe36f08aF75F278dbf5F8c301fB30EA56"],
+    "title": "DXdao Smart Vault Bridger",
+    "description": "This Smart Vault detects token balances and bridge it directly to the Smart Vault in Goerli"
```
Comment (facuspagnuolo) [inline data/smart-vaults/dxdao-bridger-l2.json:5]:
```suggestion
    "description": "This Smart Vault detects token balances and bridge them directly to the Smart Vault in Goerli"
```

File: data/smart-vaults/dxdao-bridger-l1.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+    "networks": [5],
+    "addresses": ["0x282d45DFe36f08aF75F278dbf5F8c301fB30EA56"],
+    "title": "DXdao Smart Vault Bridger",
+    "description": "This Smart Vault detects token balances and bridge it directly to the Smart Vault in mumbai."
```
Comment (facuspagnuolo) [inline data/smart-vaults/dxdao-bridger-l1.json:5]:
```suggestion
    "description": "This Smart Vault detects token balances and bridge them directly to the Smart Vault in Mumbai."
```

File: build/smart-vaults/5/0x282d45dfe36f08af75f278dbf5f8c301fb30ea56 [hunk 6]
```diff
@@ -0,0 +1 @@
+{"title":"DXdao Smart Vault Bridger","description":"This Smart Vault detects token balances and bridge them directly to the Smart Vault in Mumbai."}
```
Comment (dmf7z) [inline build/smart-vaults/5/0x282d45dfe36f08af75f278dbf5f8c301fb30ea56:1]:
```suggestion
{"title":"DXdao Smart Vault Bridger","description":"This Smart Vault detects token balances and bridges them directly to the Smart Vault in Mumbai."}
```

File: build/smart-vaults/80001/0x282d45dfe36f08af75f278dbf5f8c301fb30ea56 [hunk 7]
```diff
@@ -0,0 +1 @@
+{"title":"DXdao Smart Vault Bridger","description":"This Smart Vault detects token balances and bridge them directly to the Smart Vault in Goerli"}
```
Comment (dmf7z) [inline build/smart-vaults/80001/0x282d45dfe36f08af75f278dbf5f8c301fb30ea56:1]:
```suggestion
{"title":"DXdao Smart Vault Bridger","description":"This Smart Vault detects token balances and bridges them directly to the Smart Vault in Goerli"}
```

**Review by dmf7z — CHANGES_REQUESTED** [review]

**Pull request merged**